src/azul/json_freeze.py

```diff
@@ -27,18 +28,34 @@ def freeze(x: AnyJSON) -> AnyJSON:
 
     >>> thaw(freeze(freeze({"1":[2,3]})))
     {'1': [2, 3]}
+
+    `sequence_cls` can be used to avoid producing tuples in the output:
```

Comment:
```suggestion
    `sequence_cls` can be used to avoid producing tuples in the output. In this example we use `frozenset` which, like `tuple` is immutable, but has the potentially undesirable side-effect of eliminating duplicates and having non-deterministic ordering:
```